client/src/main/java/io/confluent/kafka/schemaregistry/ParsedSchema.java

```diff
@@ -180,7 +180,7 @@ default ParsedSchema normalize() {
    * Validates the schema and ensures all references are resolved properly.
    * Throws an exception if the schema is not valid.
    */
-  default void validate() {
+  default void validate(boolean strict) {
```

Comment:
Let's also add `default void validate()` that simply calls `validate(true)`

Then you don't have to change the test that is currently not compiling

Reply:
should we have the client side validations default to `false`? otherwise, the validation might never pass even if the server is configured with `validateFields=false`. this is also called outside tests in the maven plugin.

Reply:
Sure, `false` is fine